### PR TITLE
[ADD] Lunch: manage accounts

### PR DESCRIPTION
**Pull request by larm-odoo**
Doc needed to explain how money is added to accounts and managed.

### CODE REVIEW

**Comment by larm-odoo** [comment]
@jero-odoo this is ready for a first-round review. Thanks!

File: content/applications/hr/lunch/accounts.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+   :align: center
+   :alt: The list view of all cash moves.
+
+Add funds
+---------
+
+To add funds to a lunch account, click the :guilabel:`New` button, located in the top-left corner of
+the :guilabel:`Cash Moves` dashboard.
+
+A blank cash moves form loads. Enter the following information on the form:
```
Comment (jero-odoo) [inline content/applications/hr/lunch/accounts.rst:60]: "cash moves" form should probably have a GUI label.

File: content/applications/hr/lunch/accounts.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+- :guilabel:`Description`: enter a brief description of the transaction.
+
+.. image:: accounts/cash-move-form.png
+   :align: center
+   :alt: The cash move form filled out for a transaction of $40.00.
+
+The cash move form automatically saves as data is entered, but the form can be saved manually at any
+time by clicking the *save manually* option, represented by a :guilabel:`(cloud upload)` icon,
+located in the top-left corner of the page. To delete the record and return to the cash moves
+dashboard, click the :guilabel:`↺ (Discard changes)` icon.
```
Comment (jero-odoo) [inline content/applications/hr/lunch/accounts.rst:76]:
```suggestion
located in the top-left corner of the page. To delete the record and return to the *Cash Moves*
dashboard, click the :guilabel:`↺ (Discard changes)` icon.
```

File: content/applications/hr/lunch/accounts.rst [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+Control Accounts
+================
+
+An overview of every transaction in the *Lunch* app, including all cash deposits and purchases, can
+be viewed on the main *Control Accounts* dashboard. To access this, navigate to
+:menuselection:`Lunch app --> Manager --> Control Accounts.`
+
+All transactions are grouped :guilabel:`By Employee`, and listed in alphabetical order by the user's
```
Comment (jero-odoo) [inline content/applications/hr/lunch/accounts.rst:85]:
```suggestion
All transactions are grouped :guilabel:`By Employee`, and listed alphabetically by the user's
```

**Review by jero-odoo — APPROVED** [review]
Hi @larm-odoo great job on this one. Just a few notes. 
Also, as we discussed, the images are not loading (and prompted errors when I ran Make Fast). It seems to be because the content file for them is capitalized (Accounts) and in the doc, it is lowercase. If you change the file name it should be fine. Let me know if you have any questions, approving now :+1:

**Comment by larm-odoo** [comment]
Thanks for the review, @jero-odoo! I tried twice to fix that folder name to lower case, but it keeps reverting for some reason to a capital letter. So, I had to change the folder name so I could make sure it took (which made me rename the rst file, too).

**Comment by larm-odoo** [comment]
This is ready for a final review, @ksc-odoo!

File: content/applications/hr/lunch/user-accounts.rst [hunk 4]
```diff
@@ -0,0 +1,106 @@
+:show-content:
+:show-toc:
+
+====================
+Manage user accounts
+====================
+
+In Odoo's *Lunch* application, users pay for products directly from their *Lunch* app account. For
+funds to appear in their account, a *Lunch* app manager must transfer funds into each user's
```
Comment (ksc-odoo) [inline content/applications/hr/lunch/user-accounts.rst:9]:
```suggestion
funds to appear in their account, a *Lunch* app manager **must** transfer funds into each user's
```

File: content/applications/hr/lunch/user-accounts.rst [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+account.
+
+.. important::
+   To add funds and manage user accounts, the user must have :guilabel:`Administrator` rights set
```
Comment (ksc-odoo) [inline content/applications/hr/lunch/user-accounts.rst:13]:
```suggestion
   To add funds and manage user accounts, the user must have :guilabel:`Administrator` access rights set
```

File: content/applications/hr/lunch/user-accounts.rst [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+   for the :guilabel:`Lunch` application. This is verified by navigating to :menuselection:`Settings
+   app --> → Manage Users`. Then, click on a user to view the various settings. For more information
+   on access rights, refer to the :doc:`Access rights <../../general/users/access_rights/>`
+   documentation.
```
Comment (ksc-odoo) [inline content/applications/hr/lunch/user-accounts.rst:17]:
```suggestion
   for the *Lunch* application. This is verified by navigating to :menuselection:`Settings
   app --> → Manage Users`. Then, click on a user to view their various settings and access rights. 
   
   For more information on access rights, refer to the :doc:`Access rights <../../general/users/access_rights/>`
   documentation.
```

File: content/applications/hr/lunch/user-accounts.rst [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+The *Lunch* application does **not** directly interface in any way with software or products linked
+to any monetary accounts or billing. Money **cannot** be transferred from users' bank accounts, nor
+users' credit cards be charged.
```
Comment (ksc-odoo) [inline content/applications/hr/lunch/user-accounts.rst:21]:
```suggestion
can users' credit cards be charged.
```

File: content/applications/hr/lunch/user-accounts.rst [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+     can purchase tickets from a *Lunch* app manager, who then updates the user's account.
+
+.. _lunch/cash-moves:
+
+Cash Moves
+==========
+
+To add funds to user accounts, each cash move must be individually logged. To view all cash move
+records, or create a new cash move, navigate to :menuselection:`Lunch app --> Manager --> Cash
+Moves`.
```
Comment (ksc-odoo) [inline content/applications/hr/lunch/user-accounts.rst:44]:
```suggestion
Moves`. Doing so reveals the :guilabel:`Cash Moves` dashboard.
```

File: content/applications/hr/lunch/user-accounts.rst [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+
+All cash moves are presented in a list view, displaying each record's :guilabel:`Date`,
+:guilabel:`User`, :guilabel:`Description`, and :guilabel:`Amount` for each record. The total of all
+the cash moves is displayed at the bottom of the :guilabel:`Amount` column.
```
Comment (ksc-odoo) [inline content/applications/hr/lunch/user-accounts.rst:48]:
```suggestion
On the :guilabel:`Cash Moves` dashboard, all cash moves are presented in a default list view, displaying each record's :guilabel:`Date`, :guilabel:`User`, :guilabel:`Description`, and :guilabel:`Amount`. The total of all
the cash moves is displayed at the bottom of the :guilabel:`Amount` column.
```

File: content/applications/hr/lunch/user-accounts.rst [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+Add funds
+---------
+
+To add funds to a lunch account, click the :guilabel:`New` button, located in the top-left corner of
+the :guilabel:`Cash Moves` dashboard.
+
+A blank :guilabel:`Cash Moves` form loads. Enter the following information on the form:
+
+- :guilabel:`User`: select the user depositing cash into their account from the drop-down menu. If
+  the user is not in the database, they can be created by clicking either :guilabel:`Create "user"`
```
Comment (ksc-odoo) [inline content/applications/hr/lunch/user-accounts.rst:63]:
```suggestion
  the user is not in the database, they can be created by typing their name in the :guilabel:`User` field, and clicking either :guilabel:`Create "user"`
```

File: content/applications/hr/lunch/user-accounts.rst [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+  or :guilabel:`Create and edit...` to create the user and edit the :guilabel:`Create User` form.
```
Comment (ksc-odoo) [inline content/applications/hr/lunch/user-accounts.rst:64]:
```suggestion
  or :guilabel:`Create and edit...` to create the user and edit their :guilabel:`Create User` form.
```

File: content/applications/hr/lunch/user-accounts.rst [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+- :guilabel:`Description`: enter a brief description of the transaction.
+
+.. image:: user-accounts/cash-move-form.png
+   :align: center
+   :alt: The cash move form filled out for a transaction of $40.00.
+
+The cash move form automatically saves as data is entered, but the form can be saved manually at any
+time by clicking the *save manually* option, represented by a :guilabel:`(cloud upload)` icon,
+located in the top-left corner of the page. To delete the record and return to the *Cash Moves*
+dashboard, click the :guilabel:`↺ (Discard changes)` icon.
```
Comment (ksc-odoo) [inline content/applications/hr/lunch/user-accounts.rst:76]: I'm suggesting removing this portion, as this basic level of functionality is fairly rudimentary, and present on almost every page throughout the database. Also, I'm sure there's a 'Getting Started' doc that explains the functionality/purpose of those buttons. But I genuinely appreciate your attention to detail and desire to paint a complete picture for the reader -- however, in this case, I personally don't think it's super necessary.
```suggestion

```

File: content/applications/hr/lunch/user-accounts.rst [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+
+Control Accounts
+================
+
+An overview of every transaction in the *Lunch* app, including all cash deposits and purchases, can
+be viewed on the main *Control Accounts* dashboard. To access this, navigate to
```
Comment (ksc-odoo) [inline content/applications/hr/lunch/user-accounts.rst:82]:
```suggestion
be viewed on the main *Control Accounts* dashboard. To access this dashboard, navigate to
```

File: content/applications/hr/lunch/user-accounts.rst [hunk 14]
```diff
@@ -0,0 +1,106 @@
...
+:menuselection:`Lunch app --> Manager --> Control Accounts.`
+
+All transactions are grouped :guilabel:`By Employee`, and listed alphabetically by the user's first
+name. At the end of the user's name, a number appears. This indicates the number of individual
+records logged for that user.
+
+The default view is to have all individual transactions hidden. To view all transactions for a user,
+click the :guilabel:`▶ (triangle)` icon at the end of the name to expand the group.
```
Comment (ksc-odoo) [inline content/applications/hr/lunch/user-accounts.rst:90]:
```suggestion
click the :guilabel:`▶ (triangle)` icon to the left of the desired name to expand that specific group.
```

Comment (ksc-odoo) [reply content/applications/hr/lunch/user-accounts.rst:90]: Can't you also reveal the tucked list by clicking on the user name? If so, you may want to consider adding that in...just a suggestion

File: content/applications/hr/lunch/user-accounts.rst [hunk 15]
```diff
@@ -0,0 +1,106 @@
...
+
+Each record includes the :guilabel:`Date`, :guilabel:`User`, :guilabel:`Description`, and
+:guilabel:`Amount`.
+
+.. image:: user-accounts/control-accounts.png
+   :align: center
+   :alt: The Control Accounts dashboard with two employee's transactions expanded.
+
+.. important::
+   This list only displays the various transactions within the *Lunch* app and does **not** allow
```
Comment (ksc-odoo) [inline content/applications/hr/lunch/user-accounts.rst:100]:
```suggestion
   This list only displays the various transactions within the *Lunch* app, and does **not** allow
```

File: content/applications/hr/lunch/user-accounts.rst [hunk 16]
```diff
@@ -0,0 +1,106 @@
...
+   modifications to be made to any records listed.
+
+   Cash moves can be modified, but **only** from the :ref:`cash moves <lunch/cash-moves>`
```
Comment (ksc-odoo) [inline content/applications/hr/lunch/user-accounts.rst:103]:
```suggestion
   Cash moves can be modified, but **only** from the :ref:`Cash Moves <lunch/cash-moves>`
```

**Review by ksc-odoo — APPROVED** [review]
Great work on this PR, @larm-odoo -- just finished my final review. Once you implement the necessary changes left in my feedback (all of which are super minor), you can tag this one for Tech Review. Thanks! 👍

**Comment by StraubCreative** [comment]
- Final review CRs implemented on [6cd6870](https://github.com/odoo/documentation/commit/6cd68702213d510e3be212635a9362c8b2e62ab8)
- CI checks pass as of [a6a3dd0](https://github.com/odoo/documentation/pull/8700/commits/a6a3dd0385c631b03af943d002a798a76ead7b11) ✅

**Review by StraubCreative — APPROVED** [review]
Great doc @larm-odoo, short and sweet! 👍 
.....
@robodoo r+

**Pull request closed without merging**